tests/manage/noobaa/conftest.py

```diff
@@ -44,32 +44,44 @@ def object_cleanup():
                 command=craft_s3_command(noobaa_obj, "rm " + uploaded_filename),
                 secrets=[noobaa_obj.access_key_id, noobaa_obj.access_key, noobaa_obj.endpoint]
             )
+
     request.addfinalizer(object_cleanup)
     return uploaded_objects_paths
 
 
 @pytest.fixture()
-def created_buckets(request, noobaa_obj):
+def created_buckets(request, noobaa_obj, amount=1):
```

Comment:
Since this is a fixture, amount arg should not be passed to this function like it was a regular function.
suggesting maybe to change this fixture to be in the factory design. Then you'll be able to call it from within the test or from the test setup and pass it args.
See the usage of tests/conftest.py/pod_factory fixture as a reference